src/js/handler/time/core.js

```diff
@@ -89,7 +89,7 @@ var timeCore = {
          * @param {number} hourStart Can limit of render hour start.
          * @returns {object} - common event data for time.*
          */
-        return util.bind(function(startDate, endDate, hourStart) {
+        return function(startDate, endDate, hourStart) {
```

Comment:
`_retriveScheduleDataFromDate` doesn't need to return a function. You will see it if find all occurrences. `_retriveScheduleDataFromDate` needs refactoring.